javascript/javascript1/week4/fibnum.js

```diff
@@ -0,0 +1,15 @@
+let n1 = 0;
+let n2 = 1;
+
+function fib(n) {
+   let fibNum = [n1, n2];
```

Comment:
This can be `const` and if you follow the previous comment, skip making variables for 0 and 1 and just have it directly here:
```suggestion
   const fibNum = [0, 1];
```